README.md

```diff
@@ -208,6 +217,28 @@ Uses of `oBrandGet` within `oBrandConfigureFor` [retrieves a variable for a vari
 }
 ```
 
+### Override The Current Brand
+
+It may be desirable to output a customised variant of a component based on brand configuration. To do this create a component specific mixin which maps arguments to `oBrandOverride`. This prevents brand configuration from becoming a public interface, which makes it possible to change brand variables at a later point without a breaking change.
+
+The following contrived example shows how to override brand variables:
+
+```scss
+@mixin oExample($background-color, $forground-color) {
```

Comment:
Perhaps this functionality would be more clear if we named the contrived mixin something bordering on painfully explicit, like `@mixin oExampleSpecifyCustomizedAccentColors`? (I know, sooper excessive, but mostly just trying to illustrate a point)

Reply:
_Maybe_ not quite that ;) but yeah good idea 👍